src/components/buttonLink/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from 'react';
+import {
+  Button as ChakraButton,
+  ButtonProps as ChakraButtonProps,
+} from '@chakra-ui/react';
+
+type SharedProps = {
+  variant?: 'primary' | 'secondary';
+  size?: 'md' | 'lg';
+  url: string;
+  isExternal?: boolean;
+  rel?: string;
+} & Pick<ChakraButtonProps, 'children'>;
+
+export type ButtonProps = SharedProps;
+
+const Button: FC<ButtonProps> = ({
```

Comment:
maybe personal preference but I would not create a separate component. Typescript should allow us to expect different props if you choose the "link" variant. What do you think @Averethel